aten/src/ATen/native/cpu/zmath.h

```diff
@@ -79,22 +64,22 @@ constexpr VALUE_TYPE real_impl (SCALAR_TYPE z) {
 }
 
 template<>
-constexpr std::complex<float> real_impl <std::complex<float>> (std::complex<float> z) {
-  return std::complex<float>(z.real(), 0.0);
+constexpr c10::complex<float> real_impl <c10::complex<float>> (c10::complex<float> z) {
+  return c10::complex<float>(z.real(), 0.0);
 }
 
 template<>
-constexpr float real_impl <std::complex<float>, float> (std::complex<float> z) {
+constexpr float real_impl <c10::complex<float>, float> (c10::complex<float> z) {
   return z.real();
 }
 
 template<>
-constexpr std::complex<double> real_impl <std::complex<double>> (std::complex<double> z) {
-  return std::complex<double>(z.real(), 0.0);
+constexpr c10::complex<double> real_impl <c10::complex<double>> (c10::complex<double> z) {
```

Comment:
Looks like this is not needed, `std::real` should work. @anjali411 Please hold on and I will let you know when this is ready.